### PR TITLE
Update AWS behavior

### LINKED ISSUE #69: Check function response from Local Emulator against response from real invocation on AWS
The response of the emulated function call should be as similar to the one the users sees when invoking the deployed function on the providers infrastructure.

The `invoke local` implementation shows what parts are missing: https://github.com/serverless/serverless/tree/master/lib/plugins/aws/invokeLocal

The code where this needs to be updated can be found in the providers middleware: https://github.com/serverless/local-emulator/blob/master/src/core-middlewares/1-aws-transform.js

/cc @brianneisler @eahefnawy

### PR DESCRIPTION
**Pull request by pmuens**
Closes #69 

/cc @eahefnawy @brianneisler @nikgraf

### CODE REVIEW

File: src/core-middlewares/1-aws-transform.js [hunk 1]
```diff
@@ -57,13 +57,50 @@ const preInvoke = (data) => {
...
+        awsRequestId: 'id',
+        invokeid: 'id',
+        logGroupName: payload.functionConfig.env.LOG_GROUP_NAME,
+        logStreamName: '2015/09/22/[HEAD]13370a84ca4ed8b77c427af260',
+        functionVersion: 'HEAD',
+        isDefaultFunctionVersion: true,
+        functionName: payload.functionConfig.env.FUNCTION_NAME,
+        memoryLimitInMB: '1024',
+        succeed(result) {
+          return Object.keys(functionParams).callback(null, result);
```
Comment (pmuens) [inline src/core-middlewares/1-aws-transform.js]: Calling the `callback` in the context methods here needs to be fixed.

File: src/core-middlewares/1-aws-transform.js [hunk 2]
```diff
@@ -57,13 +57,50 @@ const preInvoke = (data) => {
...
       event: payload.payload,
-      context: {},
-      callback: (error, result) => {
-        if (error) throw new Error(error);
-        console.log(JSON.stringify(result)); // eslint-disable-line
+      context: {
+        awsRequestId: 'id',
+        invokeid: 'id',
+        logGroupName: payload.functionConfig.env.LOG_GROUP_NAME,
+        logStreamName: '2015/09/22/[HEAD]13370a84ca4ed8b77c427af260',
```
Comment (nikgraf) [inline src/core-middlewares/1-aws-transform.js]: @pmuens should the date be dynamic here? I don't know how logStreamNames are constructed

**Comment by nikgraf** [comment]
looks good. Would be nice to have some tests for the different callback possibilities. Otherwise this can be easily missed in the future.

**Comment by pmuens** [comment]
Good catch! Thanks for the review @nikgraf! 👍

I'll revisit and finish this PR tomorrow. 

AFAIK there are some changes which needs to be applied since the behavior of AWS Lambda slightly changed over the last couple of months (there's also a PR in the Framework repo which resolves an inconsistency with the time in milliseconds).



Von meinem iPhone gesendet

File: src/core-middlewares/1-aws-transform.js (hunk 2 above)
Comment (pmuens) [inline src/core-middlewares/1-aws-transform.js]: 🤔 I copied-and-pasted that from `invokeLocal`. So not 100% sure here.

**Pull request merged**